### PR TITLE
v1.10 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
 # Mattermost Mobile Apps Changelog
 
+## v1.10.0 Release
+- Release Date: July 16, 2018
+- Server Versions Supported: Server v4.0+ is required, Self-Signed SSL Certificates are not supported
+
+### Highlights
+
+#### Channel drawer performance
+ - On Android, replaced the current drawer with DrawerLayout and brought back the channel loader when switching channels.
```
Comment (enahum) [inline CHANGELOG.md]: This was made for both Android and iOS

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+
+#### Markdown support for announcement banner
+
+### Improvements
+
+ - Added experimental support for certificate-based authentication (CBA) for iOS to identify a user or a device before granting access to Mattermost. See [documentation](https://docs.mattermost.com/deployment/certificate-based-authentication.html) to learn more.
+ - Changed post textbox to not be a connected component.
+ - Allow connecting to mattermost instances hosted at subpaths.
+ - Allow retrieving posts when push notifications are received to improve messages appearing more quickly.
+ - Implemented ``FormattedMarkdownMessage`` to format translations.
```
Comment (enahum) [inline CHANGELOG.md]: This is on Master not on 1.10

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+
+#### Channel drawer performance
+ - On Android, replaced the current drawer with DrawerLayout and brought back the channel loader when switching channels.
+
+#### Channel loading improvements by retrieving posts with push notifications
+
+#### Markdown support for announcement banner
+
+### Improvements
+
```
Comment (enahum) [inline CHANGELOG.md]: Added support for starting youtube videos at a given time

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,39 @@
...
+ - Implemented ``FormattedMarkdownMessage`` to format translations.
+
+### Bug Fixes
+
+ - Fixed an issue where the unread badge background was always white.
+ - Fixed an issue where a username repeated in system message if user was added to a channel more than once.
+ - Fixed an issue where Android Sharing from Microsoft apps failed.
+ - Fixed an issue where YouTube crashed the app if link did not have a time set.
+ - Fixed an issue where System Admins did not see all teams available to join on mobile.
+ - Fixed an issue with iOS extension crash and ability to share from Files app.
```
Comment (enahum) [inline CHANGELOG.md]: This was not a crash just unable to share from the files app

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,39 @@
 # Mattermost Mobile Apps Changelog
 
+## v1.10.0 Release
+- Release Date: July 16, 2018
+- Server Versions Supported: Server v4.0+ is required, Self-Signed SSL Certificates are not supported
+
+### Highlights
+
```
Comment (enahum) [inline CHANGELOG.md]: Support for timezone and out of office

**Comment by amyblais** [comment]
Can we merge this PR?

@esethna @enahum Please let me know if there is anything else to edit here.

**Comment by enahum** [comment]
@amyblais what about the highlights and improvements in my comments?

**Comment by amyblais** [comment]
@enahum I added all of them.

I put the timezone and out of office features also in the Improvements section because that's what was done in the server changelog when those features were added on the server side earlier.

**Comment by enahum** [comment]
IMO They should be added here as well, the changelog for the server/webapp is totally different than this one, so once the app supports those features it should be added in the app's changelog as well :)

**Comment by amyblais** [comment]
They are in the Improvements section.

They are experimental features, so when we added them to the server changelog earlier, we had decided we don't want to highlight them. But I'm 0/5 since I'm not sure if RN works the same way.

**Comment by amyblais** [comment]
No worries, I realize I wasn't super clear on my earlier post.

Thank you!

**Pull request merged**